ybacklight

```diff
@@ -79,9 +79,9 @@ sub set_percentage {
         $pct = 100;
     }
     my $bri_max = get_numeric_attribute($driver, "max_brightness");
-    my $bri_new = int($bri_max/100*$pct); 
+    my $bri_new = ($bri_max/100*$pct);
     say "Setting brightness to $bri_new/$bri_max ($pct%)" if $verbose;
-    set_attribute($driver, "brightness", $bri_new);
+    set_attribute($driver, "brightness", int($bri_new));
```

Comment:
Yes, this call to `int` is necessary, at least for the modesetting driver.  With this driver, sending a decimal/fractional/float value to `set_attribute`/the sysfs node has no effect.  They must be integers/non-decimals.

Perhaps this is actually the problem with `xbacklight` and the modesetting xf86 driver?  Maybe `xbacklight` thinks decimals are okay, and e.g. the Intel xf86 driver supports this, but the KMS/modesetting xf86 driver does not?

Regardless, `ybacklight` doesn't work without this.